app/search/controllers.py

```diff
@@ -24,8 +24,8 @@ def search_packages():
             - in: query
               name: q
               type: string
-              required: true
-              description: search query string e.g. q=query publisher=pub
+              required: false
```

Comment:
This seems an unrelated change. Why is it in this PR? What does it actually relate to?

Do you do `git diff --cached` before committing and check what will be included?